net/vpnbypass/files/vpnbypass.init

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh /etc/rc.common
 
-START=92
+START=99
```

Comment:
Is there a reason for this change of priority to 99?

All normal packages should run before 95, so that S95done runs last and runs /etc/rc.local that is supposed to be run after the normal init scripts.